client-side/src/stories/GenericInput/genericInput.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
why is this a new file? why didn't you change the existing one?
this comment is relevant for all genericInput files that you changed in the PR (scss, js, stories...)